src/main/resources/db/migration/V8__create_project_delivery_employee_table.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE projectDeliveryEmployees (
+    projectId int NOT NULL AUTO_INCREMENT,
+    deliveryEmployeeId int NOT NULL AUTO_INCREMENT,
+    startDate DateTime NOT NULL TIMESTAMP,
+    finishDate DateTime NOT NULL,
```

Comment:
should this be in this table? start date and finish date is unique to the project unless its an employee start date in which case it has an incorrect name, Also why timestamp?